src/main/kotlin/no/nav/familie/baks/mottak/config/featureToggle/FeatureToggleConfig.kt

```diff
@@ -6,5 +6,6 @@ class FeatureToggleConfig {
         const val AUTOMATISK_JOURNALFØRING_AV_KONTANTSTØTTE_SØKNADER = "familie-baks-mottak.automatisk-journalforing-av-ks-soknad"
         const val AUTOMATISK_JOURNALFØRING_AV_BARNETRYGD_SØKNADER = "familie-baks-mottak.automatisk-journalforing-av-ba-soknad"
         const val HOPP_OVER_INFOTRYGD_SJEKK = "familie-ba-sak.hopp-over-infotrygd-sjekk"
+        const val BRUK_ENHETSNUMMERSERVICE = "familie-baks-mottak.bruk-enhetsnummerservice"
```

Comment:
```suggestion
        // Release
        const val AUTOMATISK_JOURNALFØRING_AV_BARNETRYGD_SØKNADER = "familie-baks-mottak.automatisk-journalforing-av-ba-soknad"
        const val AUTOMATISK_JOURNALFØRING_AV_KONTANTSTØTTE_SØKNADER = "familie-baks-mottak.automatisk-journalforing-av-ks-soknad"
        const val BRUK_ENHETSNUMMERSERVICE = "familie-baks-mottak.bruk-enhetsnummerservice"
```

Automatisk journalføring av ba og ks + enhetsnummer er egentlig ikke operasjonelle.
Vi burde kanskje flytte dem ned til egen //release?